tests/infra/network.py

```diff
@@ -736,22 +736,20 @@ def trust_node(
         try:
             if self.status is ServiceStatus.OPEN:
                 valid_from = valid_from or datetime.now()
-                timeout = ceil(args.join_timer_s * 2)
+                # Note: Timeout is function of the ledger size here since
```

Comment:
Changing some of the timeout logic here that was actually silently affected by previous changes (e.g. only opening the RPC interface when the service open transaction has been committed, rather than as soon as we've joined).